index.js

```diff
@@ -75,6 +77,8 @@ module.exports = function prettyFactory (options) {
       return
     }
 
+    const prettifiedMessage = prettifyMessage({ log, messageKey, colorizer, messageFormat })
```

Comment:
Why does this action happen before ignoring keys now?

Reply:
Support to a log property can be used by `messageFormat` and `ignore` at the same time.